openedx/core/lib/request_utils.py

```diff
@@ -220,7 +218,7 @@ def _get_expected_error_settings_dict():
     if _EXPECTED_ERROR_SETTINGS_DICT is not None:
```

Comment:
1. `_EXPECTED_ERROR_SETTINGS_DICT` contains the word `expected`, and should be switched to `_IGNORED_ERROR_SETTINGS_DICT`.
```
2. Please search for all occurrences of `expected`. Most will change to `ignored`. Some might be in code that gets deleted.

Reply:
yes, I am working on it.